c2cgeoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl

```diff
@@ -197,18 +197,27 @@ Version 1.6
 18. In your `apache/wsgi.conf.mako` or `apache/wsgi.conf.in` file replace all the
     `apache-entry-point` by `apache_entry_point`
 
-19. In the `{{package}}/subscribers.py`, `{{package}}/model.py`, `{{package}}/form.py` do the following changes:
+19. In your `apache/frontend.conf.mako`or `apache/frontend.conf.in` file in the
```

Comment:
no ... inligning => "... in the `<LocationMatch /${instanceid}/wsgi/>` add the ..."